arches/app/models/graph.py

```diff
@@ -1307,7 +1307,7 @@ def serialize(self, fields=None, exclude=None, force_recalculation=False):
         exclude = [] if exclude is None else exclude
 
         if self.publication and not force_recalculation:
-            deserialized_graph = JSONDeserializer().deserialize(self.publication.serialized_graph)
+            deserialized_graph = self.publication.serialized_graph
```

Comment:
nit: can `deserialized_graph` be changed to something else?  `deserialized_graph = ...serialized_graph` is confusing

Reply:
yep it is.